src/components/Settings/SubscriptionsSettingsCards.vue

```diff
@@ -10,7 +10,7 @@
 
 		<!-- Monthly Good Settings -->
 		<subscriptions-monthly-good
-			v-if="!isOnetime && !hasModernSub"
+			v-if="!isOnetime && (!hasModernSub || hasActiveMonthlyGoodSubscription)"
```

Comment:
Another approach would be to replace `!hasModernSub` with `! hasActiveCauseSubscription` and that would allow them through too without the extra logic added below.

Reply:
Your suggestion is quite a bit cleaner so I'll update for that. Thanks!